amishasao.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "@amishasao"
+project: "RoboSense"
+---
+
+# Project Name: RoboSense
+
+## Summary
+
+What are you going to build? What does it do?
+I am going to be building a soft robotic glove for individuals with limited hand motor function to improve functionality faster and be able to grip, hold, and release objects with ease. This could be a potentially helpful tool for stroke and other patients attempting to gain back their motor function via therapy or rehabilitation. The glove itself is going to do two main functions: sense pressure (from the fingertips and wrist) and rotate the appendages (based on the pressure reading). So, for example, if I wanted to pick up a book from my desk, I would get to the book, put pressure on my wrist (which will rotate the glove and, thus, my hand downward), put pressure on my fingertips to try to hold the book, hold the book (given the glove's grip in place), and realease the pressure from my fingertips when wanting to drop the book.
+
+Why are you excited to build it?
+I am excited to build this because I believe that it can have real life implications and become a cost-effective option for patients to regain their hand motor function. Additionally, I am really excited to 3d print nylon and experiment with haptic sensors judging the movement of the wearable glove.
+
+## Plan
+
+What steps are you going to take to build it? What will you do first, second, third, etc.? What tools are you going to use?
+1. 3d model the basic structure of the backbone of the moving parts of the glove
+2. 3d model the basic structure of the nylon outer covering
+3. slice and 3d print both components
+4. use an arduino nano & 6 servo motors to guide the movement of the backbone of the glove
+5. attach haptic sensors to sense the pressure live
+6. program the arduino to connect the servo motor motion with the pressure sensor live data
+7. attach the nylon covering around the backbone
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
+| 6 haptic sensors   | https://www.aliexpress.us/item/2251832669872365.html?spm=a2g0o.ppclist.product.10.1a6eW5W6W5W6Ff&pdp_npi=2%40dis%21USD%21US%20%243.37%21%241.79%21%21%21%21%21%4021032fa016724082409855056ebc66%2165296889214%21btf&_t=pvid%3A43deb1cb-b818-4216-8c3b-dff0952b8e32&afTraceInfo=32856187117__pc__pcBridgePPC__xxxxxx__1672408241&gatewayAdapt=glo2usa&_randl_shipto=US       | $12.69  |
```

Comment:
Will you be needing some wire and/or a soldering kit to connect the sensors to the Arduino?

Reply:
Yes, I think so. I'll try to find a suitable one asap!

Reply:
Yes, let me check which one I need.